scripts/zones/Beaucedine_Glacier/mobs/Humbaba.lua

```diff
@@ -1,10 +1,48 @@
 -----------------------------------
 -- Area: Beaucedine Glacier
---  MOB: Humbaba
+--   NM: Humbaba
 -----------------------------------
-require("scripts/globals/regimes")
+require("scripts/globals/status")
+require("scripts/globals/magic")
 -----------------------------------
 
+function onMobInitialize(mob)
+    mob:setMobMod(dsp.mobMod.AUTO_SPIKES, mob:getShortID())
```

Comment:
```lua
mob:setMobMod(dsp.mobMod.AUTO_SPIKES, 1)
```
the short ID thing I dunno what was intended and it was never used, so I stopped doing it and plan to remove all instances of it